### PR TITLE
feat: support full match option

### LINKED ISSUE #49: Is it more appropriate to use full match than partial match in some positions?
Using partial matches in some items of the config file may cause problems.

For example, when I make the following settings:
```yaml
method: 余额
methodAccount: Assets:Alipay:Balance
```
It will also match this record, which means you use two payment accounts:
```yaml
...
method: "招商银行储蓄卡(8888)&余额"
...
```
This will result in only one payment account being generated and will not be marked with FIXME, so you may have difficulty finding errors.

So should we consider using full matches instead of partial matches in some items?

### PR DESCRIPTION
**Pull request by z2z23n0**
resolves #49

### CODE REVIEW

File: example/huobi/config.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
-defaultCashAccount: Assets:Huobi:Cash
-defaultPositionAccount: Assets:Huobi:Positions
-defaultCommissionAccount: Expenses:Huobi:Commission
-defaultPnlAccount: Income:Huobi:PnL
```
Comment (Triple-Z) [inline example/huobi/config.yaml:4]: Huobi's default accounts should not be `FIXME`, these default accounts are actually used in the transactions.

File: example/huobi/example-huobi-output.beancount [hunk 2]
```diff
@@ -1,13 +1,13 @@
 option "title" "测试"
 option "operating_currency" "CNY"
 
-1970-01-01 open Assets:Huobi:Cash
-1970-01-01 open Assets:Huobi:Positions
+1970-01-01 open Assets:FIXME
+1970-01-01 open Assets:Position:FIXME
```
Comment (Triple-Z) [inline example/huobi/example-huobi-output.beancount:5]: Huobi's default accounts should not be FIXME, these default accounts are actually used in the transactions.

**Review by Triple-Z — CHANGES_REQUESTED** [review]

File: example/huobi/example-huobi-output.beancount (hunk 2 above)
Comment (z2z23n0) [reply example/huobi/example-huobi-output.beancount:5]: OK, I will revert that.

**Review by gaocegege — APPROVED** [review]
/cc @Triple-Z 

LGTM

**Review by Triple-Z — APPROVED** [review]
LGTM

**Comment by Triple-Z** [comment]
Thanks for your contribution🎉

**Pull request merged**